### ISSUE TITLE
WebSocket interface sending incorrect InventoryFull flag

### ISSUE THREAD

**Issue by GediminasMasaitis** [PocketMobsters/MobBotTemp#113]
![image](https://cloud.githubusercontent.com/assets/7861188/17747703/a4130254-64be-11e6-896c-ab0bcb3c4947.png)

I've confirmed that the inventory is full, pokestop visit gets no items, yet `InventoryFull` is set to false. 

I'll fix this myself soon, just posting here in case I forget

**Comment by XeonPowder** [comment]
Line: 392 of FarmPokeStopTask.cs
https://github.com/PocketMobsters/MobBotTemp/blob/development/PoGo.PokeMobBot.Logic/Tasks/FarmPokestopsTask.cs#L392
When it's full it recycles, so i guess it'll never be full.
unless you are banned :palm_tree: :cocktail:

**Comment by FabryB** [comment]
It depends on settings: if you write in config to keep 5000 balls, for sure it would get full

**Comment by XeonPowder** [comment]
i thought we added a failsafe so this wouldnt happen

**Comment by FabryB** [comment]
No, that is not the failsafe... that will TRY to clean your inventory, but if your KEEP options are wrong it won't still recycle anything...
A real failsafe would be throw an error if sum of all you want to keep >= inventory space (not static 350 since you can buy more, even if not probable with botted accounts)
